packages/scene-composer/src/utils/entityModelUtils/nodeComponent.ts

```diff
@@ -80,8 +84,28 @@ export const createNodeEntityComponent = (node: ISceneNode, layerId?: string): C
   return comp;
 };
 
-export const updateNodeEntityComponent = (node: ISceneNode, layerId?: string): ComponentUpdateRequest => {
+export const updateNodeEntityComponent = (
+  node: ISceneNode,
+  layerId?: string,
+  updateType?: ComponentUpdateType,
+): ComponentUpdateRequest => {
   const request = createNodeEntityComponent(node, layerId);
+  const entityBinding = node.components?.find((component) => component.type === 'EntityBinding');
+  const entityId = (entityBinding as IEntityBindingComponent)?.valueDataBinding?.dataBindingContext?.entityId;
+
+  if (entityId) {
+    request.properties![DEFAULT_ENTITY_BINDING_RELATIONSHIP_NAME] = {
+      value: {
+        relationshipValue: {
+          targetEntityId: entityId,
+        },
+      },
+    };
+  } else if (updateType === 'DELETE') {
+    // TODO: handle clear entity binding
+    console.error('Delete Entity Binding is not currently supported.');
```

Comment:
Nit-pick: If this is intended to let users know that something was improperly configured we should have the messaging localized.

Reply:
this is temporary. we want to support delete entity binding by GA